ClientServer.go

```diff
@@ -47,7 +46,10 @@ func Server() {
 		// send new string back to client
 		_, sendingerror := conn.Write([]byte(newmessage + "\n"))
```

Comment:
you can use `err`, previous value of error is not useful because it already passed through  `if` condition.